spec/plugin.integration.spec.js

```diff
@@ -436,3 +436,51 @@ describe('ManifestPlugin with memory-fs', function() {
     });
   });
 });
+
+describe('scoped hoisting', function() {
+  var compiler;
+  var hashes;
+
+  beforeAll(function () {
+    fse.outputFileSync(path.join(__dirname, 'output/scoped-hoisting/index.js'), 'import { ReactComponent } from "./logo.svg";');
+    fse.outputFileSync(path.join(__dirname, 'output/scoped-hoisting/logo.svg'), '<svg />');
+
+    hashes = [];
+  });
+
+  afterAll(() => {
+    compiler.close()
+  })
+
+  it('outputs a manifest', function(done) {
+    const plugins = [
+      new ManifestPlugin(),
+    ];
+    if (webpack.optimize.ModuleConcatenationPlugin) {
```

Comment:
Can you use `isWebpack4()` so it's easy to refactor out once we drop webpack < 4